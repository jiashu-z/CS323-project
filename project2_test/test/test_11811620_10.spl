struct Child
{
  int name;
  int age;
}var1, arr1[1], arr1[1][1], arr3[1][1][1];

int test_11811535_9(){
  struct Child a;
  struct Child var1;
  struct Child arr3[1];
  arr3[0][0][0] = var1;
  return 0;
}

