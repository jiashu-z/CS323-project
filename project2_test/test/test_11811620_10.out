Error type 3 at Line 5: redefine variable: arr1
Error type 3 at Line 9: redefine variable: var1
Error type 3 at Line 10: redefine variable: arr3
